### ISSUE TITLE
About the absent annotation of training set

### ISSUE THREAD

**Issue by suyeH** [wangxiao5791509/TNL2K_evaluation_toolkit#10]
Hi, thanks for your work. I download your dataset from OneDrive and find that there is no absent and attribution annotation for the training set, which is different from your description in the paper. Is this expected?
![image](https://user-images.githubusercontent.com/23137816/145678286-99b6f3a2-b5ca-4345-8760-ef62397cb36c.png)

**Comment by wangxiao5791509** [comment]
@suyeH   Hi, the language annotation is placed in the dataset folders. The absent labels can be found in this github. All the things have been released. To the best of my knowledge, some researchers have successfully developed their own language-based trackers based on this dataset. Good luck!

**Comment by suyeH** [comment]
@wangxiao5791509  Thanks for your reply. 

> The absent labels can be found in this github.

Do you mean this [annos.tar.gz](https://github.com/wangxiao5791509/TNL2K_evaluation_toolkit/blob/f68d206285e7082d51e4dbb2ab30423db3e83984/annos/annos.tar.gz) file or do I miss something? Since this file only contain the absent labels for the test set.

**Comment by wangxiao5791509** [comment]
@suyeH  Hi, sorry for the misunderstanding of your question, currently, we only have the absent label for the testing subset. If you think the training set also needs such labels, we can provide them in the journal extension (it needs some time).
